orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/WaitForDestroyedServerGroupTask.groovy

```diff
@@ -59,10 +59,12 @@ class WaitForDestroyedServerGroupTask extends AbstractCloudProviderAwareTask imp
       if (!cluster || !cluster.serverGroups) {
         return new DefaultTaskResult(ExecutionStatus.SUCCEEDED)
       }
-      if (!cluster.serverGroups.find { it.name == serverGroupName && it.region == serverGroupRegion }) {
+      def serverGroup = cluster.serverGroups.find { it.name == serverGroupName && it.region == serverGroupRegion }
+      if (!serverGroup) {
         return new DefaultTaskResult(ExecutionStatus.SUCCEEDED)
       }
-
+      def instances = serverGroup.instances?.findResults { it.name }?.join(',') ?: '[none]'
+      log.info("${serverGroupName}: not yet destroyed, found instances: ${instances}")
       return new DefaultTaskResult(ExecutionStatus.RUNNING)
```

Comment:
Good call. I was going for a minimal amount of code with some easy grep-able pattern to trace what was happening. I'll add them to the context, but leave the log messages in place so we can see the history of the instances going away by diving into logs.